codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,13 +175,19 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
+        FlowState state;
```

Comment:
@Scottmitch hmm ... really? I have never heard of this. javac will issue a warning?